osm2streets-java/src/PaintArea.java

```diff
@@ -0,0 +1,15 @@
+package org.osm2streets;
+
+import org.osm2streets.LatLon;
+
+import java.util.List;
+
+public class PaintArea {
+    public List<LatLon> area;
+    public String color;
+
+    public PaintArea(List<LatLon> area, String color) {
```

Comment:
Maybe a comment to say if this needs to be a closed polygon (first = last point or not?)

Reply:
Good question :) I'm shunting across a `goe::Polygon` outer ring for an "area", and blindly passing all the points to `g.fillPolygon(...)`.